.ado/jobs/universal.yml

```diff
@@ -10,13 +10,10 @@
       default:
         - BuildEnvironment: PullRequest
           Matrix:
-            - Name: ArmRelease
-              BuildConfiguration: Release
-              BuildPlatform: ARM
             - Name: X64Release
               BuildConfiguration: Release
               BuildPlatform: X64
-            - Name: X64Debug
+            - Name: X86Debug
```

Comment:
we should run 64 bit because a) this is what our customers will be running, b) moving to x86 means we won't get type-narrowing errors/warnings in this pipeline

Reply:
This was x86, just labeled wrong :)

Reply:
so are you saying x64 debug & x86 release?

Reply:
I want at least coverage for x86/x64 and debug/release. I'm a bit ambivalent of which is which. And if x64 is more important I'd like to argue that the release version is more important for that, hence the current split:

Reply:
got it. having x64 release makes sense and we still want coverage for debug and x86 so x86debug is ok, I had missed the existing typo